app/bundles/CoreBundle/Loader/ParameterLoader.php

```diff
@@ -76,7 +77,12 @@ public function loadIntoEnvironment()
         MessengerEnvLoader::load($this->parameterBag, $defaultParameters, $envVariables);
 
         // Load the values into the environment for cache use
-        $dotenv = new \Symfony\Component\Dotenv\Dotenv();
+        $dotenv = new Dotenv();
```

Comment:
This won't mute this warning though, right?
```
 The default value of "$usePutenv" argument of "Symfony\Component\Dotenv\Dotenv::__construct" will be changed from "true" to "false" in Symfony 5.0. You should define its value explicitly.
 ```
```suggestion
        $dotenv = new Dotenv(false);
```